cluster/test-deploy/gcp-crio/vars-origin.yaml

```diff
@@ -44,7 +44,7 @@ openshift_schedulable: True
 openshift_hosted_registry_storage_gcs_keyfile: "{{ files_dir }}/gce.json"
 openshift_hosted_registry_storage_kind: object
 openshift_hosted_registry_storage_provider: gcs
-openshift_hosted_registry_storage_gcs_bucket: origin-ci-test-{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}
```

Comment:
There is only one `test-platform-results` bucket. `INSTANCE_PREFIX` appears to be allow access to other buckets that start with `origin-ci-test-` . Those are private and we don't need to migrate them.